File: ZMusic-Plugin/src/main/java/me/zhenxin/zmusic/command/CmdBC.java
package me.zhenxin.zmusic.command;

import net.md_5.bungee.api.CommandSender;
import net.md_5.bungee.api.plugin.Command;
import net.md_5.bungee.api.plugin.TabExecutor;

public class CmdBC extends Command implements TabExecutor {

    public CmdBC() {
        super("zm", null, "zmusic", "music");
    }

    @Override
    public void execute(CommandSender sender, String[] args) {
        Cmd.cmd(sender, args);
    }

    @Override
    public Iterable<String> onTabComplete(CommandSender sender, String[] args) {
        return Cmd.tab(sender, args);
    }
}

File: src/main/kotlin/me/zhenxin/zmusic/command/CmdBC.kt
package me.zhenxin.zmusic.command

import me.zhenxin.zmusic.module.sender.SenderBC
import net.md_5.bungee.api.CommandSender
import net.md_5.bungee.api.plugin.Command
import net.md_5.bungee.api.plugin.TabExecutor

class CmdBC : Command("zm", null, "zmusic", "music"), TabExecutor {
    override fun execute(sender: CommandSender, args: Array<String>) {
        CmdEx.cmd(SenderBC(sender), args)
    }

    override fun onTabComplete(sender: CommandSender, args: Array<String>): Iterable<String> {
        return CmdEx.tab(SenderBC(sender), args)
    }
}
